Public/Src/Pips/Dll/Artifacts/PipArtifacts.cs

```diff
@@ -49,9 +49,45 @@ public static bool CanProduceOutputs(PipType pipType)
         /// <summary>
         /// Checks if a pip can preserved its outputs.

```

Comment:
>Checks if a pip can preserved its outputs. [](start = 12, length = 42)

Update comment to reflect this is now by file

Reply:
added remarks as well

---
In reply to: [287428462](https://github.com/microsoft/BuildXL/pull/369#discussion_r287428462) [](ancestors = 287428462)